packages/modules/src/utils/Types.ts

```diff
@@ -1,24 +1,31 @@
 import { ChainId } from '@biconomy/core-types'
 import { Signer } from 'ethers'
 
+export type ModuleVersion = 'V1_0_0' | 'V1_0_1'
+
```

Comment:
will we have multiple version right now? it should just be `V1_0_0`

Reply:
right now we will just have 1_0_0.
also versioning number should be similar for all modules right? no need to keep version enums per module.

Reply:
yeah can remove the enum part, dont't like how users has to import type and the pass it while it can be just optional string

Reply:
regarding the version of modules,
I think we can have a situation in future we might want to update a single module version.